graphql_compiler/tests/snapshot_tests/test_query_pagination.py

```diff
@@ -45,7 +46,7 @@ def test_pagination_planning_basic(self):
         query_ast = safe_parse_graphql(query)
         pagination_plan = get_pagination_plan(schema_info, query_ast, number_of_pages)
         expected_plan = PaginationPlan([
-            VertexPartition(['Animal'], number_of_pages)
+            VertexPartition(('Animal'), number_of_pages)
```

Comment:
Since this is a tuple shouldn't it'd be `('Animal',)`?

Reply:
good catch